### DISCUSSION TITLE
Have you typed 'make' in [abb_robot_msgs]?

### DISCUSSION THREAD

**Discussion by charmingotm** [ros-industrial/abb_robot_driver#102] [Q&A]
As you have seen,I got an error when I used command "rosservice call /rws/set_io_signal",and the error is "ERROR: Unable to load type [abb_robot_msgs/SetIOSignal].Have you typed 'make' in [abb_robot_msgs]?".I don't know what dose it mean and what step should I take to fix that.

**Comment by chastjj** [comment]
I have no problem with this command

**Reply by zp2546265641** [reply]
rosservice call /rws/set_io_signal "signal: 'Local_IO_1_AO1'
value: '0'" 
result_code: 2
message: "\"Service failed (e.g. timeout or resource not found, see the DEBUG log for details)\""
rosservice call /rws/get_io_signal "signal: 'Local_IO_1_AO1'" 
value: "0"
result_code: 1
message: ''
hello,i can get but i can not set? what should i do?
